src/components/styles/style.scss

```diff
@@ -39,12 +39,18 @@
   justify-content: center;
 }
 
+.select-input {
+  visibility: hidden;
+  width: 0;
+  margin-right: 10px;
+}
 .no-item-title {
   color: #ddd;
 }
 .sort-icon {
-  width: 70px !important;
-  font-size: 60px !important;
+  font-size: 50px !important;
+  border: 1px solid black;
+  border-radius: 3px;
 }
 .active-button {
   background-color: #ddd !important;
```

Comment:
try not to use "!important" if it is not necessary.

Reply:
here it is necessary because I have to disable MUI styles
so either with sx prop or with '!important'

Reply:
Ok, no problem then